### PR TITLE
Feature/show-target-speed

### PR DESCRIPTION
**Pull request by KarlSchu**
Target speed or target power is displayed right of speed Main display

### CODE REVIEW

File: docs/xbee--HC-12.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# XBee HC-12
+
+https://www.allaboutcircuits.com/projects/understanding-and-implementing-the-hc-12-wireless-transceiver-module/
+
```
Comment (andreaskuster) [inline docs/xbee--HC-12.md:4]: HC-12 is not an Xbee module to my understanding

File: include/DisplayValue.h [hunk 2]
```diff
@@ -15,10 +16,9 @@
 #include <stdlib.h>           /* abs */
 #include <typeinfo>
 
+#include <../lib/SPIBus/SPIBus.h>
```
Comment (andreaskuster) [inline include/DisplayValue.h:19]: Is there a specific reason that you are not use `<SPIBus.h>` directly? The header should be in the default include path, right?

**Review by andreaskuster — APPROVED** [review]
Looks good to me (except the comments)
One additional little remark: It´s `safety`

File: include/DisplayValue.h (hunk 2 above)
Comment (KarlSchu) [reply include/DisplayValue.h:19]: Terribly it is not found in the usual way :(
If you can fix then I am glad!

Comment (KarlSchu) [reply include/DisplayValue.h:19]: resolve for now, let's try in next round

File: docs/xbee--HC-12.md (hunk 1 above)
Comment (andreaskuster) [reply docs/xbee--HC-12.md:4]: Exactly, this is not a XBee module to my understanding...

**Pull request merged**